deploy/dockerephemeral/run.sh

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env bash
 
+set -x
+
 # run.sh should work no matter what is the current directory
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 DOCKER_FILE="$SCRIPT_DIR/docker-compose.yaml"
 
-docker-compose --file "$DOCKER_FILE" up
+docker compose --file "$DOCKER_FILE" up
```

Comment:
Can we change this to `docker-compose`?

Reply:
why?!  this breaks things for me.  (debian, i'm assuming i'll have to spend some time installing the apt sources from docker now.  but what's the point of changing this?)

Reply:
🤔 this broke the script for me on Ubuntu hehe